README.md

```diff
@@ -161,6 +161,22 @@ Retrieve summary metrics of all responses:
 delighted.metrics.retrieve();
 ```
 
+### Autopilot
+
+Retrieve existing autopilot configuration for the `email` platform:
+
+```javascript
+delighted.autopilotConfiguration.retrieve('email');
+```
+
+To interact with the autopilot person api, configure the autopilotPerson instance with the platform. You can list all people imported to Autopilot, add a new person, or delete an existing person:
+
+```javascript
+delighted.autopilotPerson.forPlatform('email').all();
+delighted.autopilotPerson.forPlatform('email').create( { person_email: 'email@example.com' } );
```

Comment:
possibly rename this to `autopilotMembership`, to match with the other client (e.g. https://github.com/delighted/delighted-ruby/pull/25/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR7)